docs/install.md

```diff
@@ -245,19 +242,33 @@ If EaseMesh infrastructure has already been installed, the add-on features could
 emctl install --only-add-on --add-ons=ShadowService
 ```
 
+### Install CoreDNS
+
+If you want to support non-Java languages, installing EaseMesh dedicated CoreDNS is a must. It's very easy to do it:
```

Comment:
non-Java Languages isn't correct, the pure spring-boot (without spring-cloud suite) application needs the CoreDNS installation also.

Reply:
Updated